spec/javascripts/social_networking/home-controller_spec.js

```diff
@@ -69,6 +69,29 @@ describe('HomeCtrl', function() {
     expect(controller.inFeedBrowseMode()).toBeTruthy();
   });
 
+  describe('#timeAgoInWords', function() {
+    var foo;
+
+    beforeEach(function() {
+      foo = {
```

Comment:
It would be more self-documenting if you named your spy something like "momentSpy". "Foo" can be a little obtuse.